examples/Batch.CorrespondingContentIDs-sample.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Namespace="Org.OData.Core.V1" Alias="Core" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Batch.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Batch.v1" Alias="Batch" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="corresponding.sample" Alias="self" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+      <EntityType Name="SalesQuotation">
+        <Property Name="id" Type="Edm.Int16" />
+      </EntityType>
+      <ComplexType Name="SalesQuotationItemToBeCopied">
+        <Property Name="product" Type="Edm.String" />
+      </ComplexType>
+      <EntityType Name="SalesOrder">
+        <NavigationProperty Name="Items" Type="Collection(self.SalesOrderItem)" />
+      </EntityType>
+      <EntityType Name="SalesOrderItem">
+        <NavigationProperty Name="Discounts" Type="Collection(self.SalesOrderItemDiscount)" />
+      </EntityType>
+      <EntityType Name="SalesOrderItemDiscount">
+        <Property Name="percent" Type="Edm.Decimal" />
+      </EntityType>
+      <Action Name="CreateSalesOrder" IsBound="true">
+        <Parameter Name="_it" Type="self.SalesQuotation" Nullable="false" />
+        <Parameter Name="items" Type="Collection(self.SalesQuotationItemToBeCopied)" />
+        <ReturnType Type="self.SalesOrder" Nullable="false" />
+        <Annotation Term="Batch.CorrespondingContentIDs">
+          <Collection>
+            <Record>
+              <Annotation Term="Core.Description" String="Correspondence between sales quotation item and sales order item" />
+              <PropertyValue Property="ParameterValue" PropertyPath="items" />
```

Comment:
where does this "items" come from? It's a PropertyPath, so I would expect somewhere a property "items" - the only thing I see in the example is a parameter.

Reply:
A property path relative to an action starts with with a parameter name or with `$ReturnType`.